content/actions/writing-workflows/choosing-what-your-workflow-does/caching-dependencies-to-speed-up-workflows.md

```diff
@@ -340,9 +340,9 @@ Users with `write` access to a repository can use the {% data variables.product.
 
 Caches have branch scope restrictions in place, which means some caches have limited usage options. For more information on cache scope restrictions, see "[AUTOTITLE](/actions/using-workflows/caching-dependencies-to-speed-up-workflows#restrictions-for-accessing-a-cache)." If caches limited to a specific branch are using a lot of storage quota, it may cause caches from the `default` branch to be created and deleted at a high frequency.
 
-For example, a repository could have many new pull requests opened, each with their own caches that are restricted to that branch. These caches could take up the majority of the cache storage for that repository. {% data reusables.actions.cache-eviction-policy %} In order to prevent cache thrashing when this happens, you can set up workflows to delete caches on a faster cadence than the cache eviction policy will. You can use the [`gh-actions-cache`](https://github.com/actions/gh-actions-cache/) CLI extension to delete caches for specific branches.
+For example, a repository could have many new pull requests opened, each with their own caches that are restricted to that branch. These caches could take up the majority of the cache storage for that repository. {% data reusables.actions.cache-eviction-policy %} In order to prevent cache thrashing when this happens, you can set up workflows to delete caches on a faster cadence than the cache eviction policy will. You can use the `gh` CLI to delete caches for specific branches.
 
-The following example workflow uses `gh-actions-cache` to delete up to 100 caches created by a branch once a pull request is closed.
+The following example workflow uses [`gh cache`](https://cli.github.com/manual/gh_cache) to delete up to 100 caches created by a branch once a pull request is closed.
```

Comment:
```suggestion
The following example workflow uses `gh cache` to delete up to 100 caches created by a branch once a pull request is closed.
```

We wouldn't use an in-line link here (https://docs.github.com/en/contributing/style-guide-and-content-model/style-guide#formatting-links), but since we mention, and link to, the GitHub CLI and its reference in the upper "Managing caches" section, we're fine to just delete this.